app/static/js/new_post.js

```diff
@@ -0,0 +1,176 @@
+$(function () {
+    // initialize datepickers
+    $('#meeting-date').datepicker({
+        dateFormat: 'mm/dd/yy',
+        maxDate: 0
+    }).keydown(function (e) {
+        // prevent keyboard input except for tab
+        if (e.keyCode !== 9) {
+            e.preventDefault();
+        }
+    });
+
+    $('#next-meeting-date').datepicker({
```

Comment:
Shouldn't we have a `minDate` of today here?

Reply:
What if you create the post 2 weeks after the original meeting date and the next meeting date is 1 week after the original meeting. Then if `minDate` is today, you would miss the chance to select the actual next meeting date. We would have to set minDate to be the original meeting date but only after the user selects it or else it would be null.

Reply:
Ok. Then let's leave it as is and see what happens